app/services/billable_metrics/aggregations/base_service.rb

```diff
@@ -43,6 +58,26 @@ def sanitized_name(property)
       def sanitized_field_name
         sanitized_name(billable_metric.field_name)
       end
+
+      def handle_in_advance_current_usage(total_aggregation)
+        if previous_event
+          BigDecimal(previous_event.metadata['max_aggregation']) >
+          BigDecimal(previous_event.metadata['current_aggregation'])
+
+          aggregation = total_aggregation -
+                        BigDecimal(previous_event.metadata['current_aggregation']) +
+                        BigDecimal(previous_event.metadata['max_aggregation'])
+
+          result.aggregation = aggregation
+          result.current_usage_units = total_aggregation
+        else
+          result.aggregation = total_aggregation
+          result.current_usage_units = total_aggregation
```

Comment:
Style/IdenticalConditionalBranches: Move `result.current_usage_units = total_aggregation` out of the conditional.